tools/wego-app-dev.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: wego-app
+  namespace: wego-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: wego-app
+    spec:
+      serviceAccountName: wego-app-service-account
+      containers:
+        - name: wego-app
+          image: ghcr.io/weaveworks/wego-app:latest
```

Comment:
Would it be possible to configure this while spinning up Tilt to replace it with my own dev image, for example, if I want to push/pull `wego-app:latest` from/to my own container registry to avoid using other people's dev work or pushing to `ghcr.io/weaveworks/wego-app:latest`? Or would I have to change this line manually?

Reply:
What @Skarlso said! Closing this to move the conversation there.